### PR TITLE
Fix test structure

### PR DESCRIPTION
**Pull request by dolcalmi**
- Move test files to match src structure
- Fix update routing fees test to make tests pass (now is idempotent)

### CODE REVIEW

File: test/integration/services/lnd/utils.spec.ts [hunk 1]
```diff
@@ -80,16 +83,37 @@ describe("lndUtils", () => {
...
+
+    const endDate = new Date(Date.now() - 864e5)
+    endDate.setUTCHours(0, 0, 0, 0)
+
+    const after = startDate.toISOString()
+    const before = endDate.toISOString()
+
+    const totalFees = (await getForwards({ lnd: lnd1, after, before })).forwards
+      .map((f) => Number(f.fee_mtokens))
+      .reduce((acc, val) => acc + val)
```
Comment (samerbuna) [inline test/integration/services/lnd/utils.spec.ts:100]: Minor nitpick but I'd generally avoid multiple scanning when possible: 

`.reduce((acc, val) => acc + Number(val.fee_mtokens))`

**Review by samerbuna — DISMISSED** [review]
LGTM

File: test/integration/services/lnd/utils.spec.ts (hunk 1 above)
Comment (dolcalmi) [reply test/integration/services/lnd/utils.spec.ts:100]: It is not possible do it in that way, in that way acc would be forward type so it crash when you try to sum the fee

Comment (dolcalmi) [reply test/integration/services/lnd/utils.spec.ts:100]: I'll check if setting initial value fixes that, but I tried that way and it didn't work

Comment (dolcalmi) [reply test/integration/services/lnd/utils.spec.ts]: Fixed https://github.com/GaloyMoney/galoy/pull/411/commits/27007f331fb86f9c96147622aa2aefcbfdfe8686 with initial value. `.reduce((acc, val) => acc + Number(val.fee_mtokens), 0)`

**Review by samerbuna — APPROVED** [review]

File: test/integration/services/lnd/utils.spec.ts [hunk 2]
```diff
@@ -80,16 +83,38 @@ describe("lndUtils", () => {
...
 
-    baseLogger.debug((await getForwards({ lnd: lnd1 })).forwards, "forwards")
-
     const date = Date.now() + 60 * 60 * 1000 * 24 * 2
     jest.spyOn(global.Date, "now").mockImplementation(() => new Date(date).valueOf())
 
+    const startDate = new Date(0)
+    startDate.setUTCHours(0, 0, 0, 0)
+
+    const endDate = new Date(Date.now() - 864e5)
```
Comment (nicolasburtey) [inline test/integration/services/lnd/utils.spec.ts:92]: what is `864e5`? magic number should probably be const

**Pull request merged**